Add old openflow port (6633) and karaf ssh port (8081)

jluhrsen opened the pull request:
Signed-off-by: Jamo Luhrsen <jluhrsen@redhat.com>

jluhrsen committed:
Add old openflow port (6633) and karaf ssh port (8081)

Signed-off-by: Jamo Luhrsen <jluhrsen@redhat.com>

jluhrsen committed:
Add old openflow port (6633) and karaf ssh port (8081)

Signed-off-by: Jamo Luhrsen <jluhrsen@redhat.com>

trozet reviewed (changes requested):
You mean 8101 not 8081, right?

jluhrsen committed:
Open additional ports for OpenDaylight services

- port 8101 is open to all karaf ssh access
- port 6633 is open as the old default openflow port that is still
  in use by upstream OpenDaylight system tests. It is still enabled
  by default from the openflowplugin.

Add old openflow port (6633) and karaf ssh port (8101)

Signed-off-by: Jamo Luhrsen <jluhrsen@redhat.com>
